std/conv.d

```diff
@@ -5024,6 +5024,48 @@ T emplace(T, Args...)(void[] chunk, auto ref Args args)
     return result;
 }
 
+/**
+Constructs an object of $(D class) type $(D T) at the location $(D obj)
+references. The constructor is passed the arguments $(D Args).
+
+$(D obj) must not be a base class or interface.
+
+This function can be $(D @trusted) if the corresponding constructor of
```

Comment:
That's not how `@trusted` is used. Templated functions have attribute inference; `@safe` is inferred if the body is completely SafeD. `@trusted` makes the compiler consider it `@safe` regardless of the body. That means it will be considered `@safe` even if the constructor does unsafe operations. Thus, this should use attribute inference, not `@trusted`.

In cases where attribute inference is desired and parts of the body must be `@trusted`, you can wrap the manually-checked operations in a `@trusted` anonymous function; see `emplaceImpl` for some examples of this.